Templates/CSharp/EdgeDriverTemplate/DotNetCore/EdgeDriverTemplateCore/ProjectTemplates/CSharp/Test/EdgeDriverTemplate/EdgeDriverSample.csproj

```diff
@@ -6,9 +6,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0" />
-    <PackageReference Include="MSTest.TestAdapter" Version="2.2.8"/>
-    <PackageReference Include="MSTest.TestFramework" Version="2.2.8"/>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.2.10"/>
```

Comment:
Could we put all versions in `eng/Version.props` and use the variable instead. This would simplify further updates.